doc/user/OPTIONS.md

```diff
@@ -18,6 +18,7 @@ Options|Default|Description
 \-\-watch||Watch current file and automatically reload it whenever it is modified on disk.
 \-\-load-plugins=\<paths or names\>||List of plugins to load separated with a comma. Official plugins are `alembic`, `assimp`, `draco`, `exodus`, `occt`, `usd`, `vdb`. See [usage](USAGE.md) for more info.
 \-\-scan-plugins||Scan standard directories for plugins and display their names, results may be incomplete. See [usage](USAGE.md) for more info.
+\-\-screenshot-filename||[Screenshot](INTERACTIONS.md#taking-screenshots) filename template. Can use [template variables](#filename-templating).
```

Comment:
```suggestion
\-\-screenshot-filename||[Screenshot](INTERACTIONS.md#taking-screenshots) filename template. Can use [template variables](#filename-templating).
```

Reply:
do we need "Path to a filename to save screenshots to"? or can we leave it at "filename to save screenshots to"?

Reply:
fine by me